Bottom: 0d51a85de97c601609590dcc7326e6da10b941a8
Top:    5f171fa19953f9350c60baf9a3a30bf33c41bbbe
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-06 10:28:29 +0800

Refresh of autofs-5.0.7-dont-fail-on-master-map-self-include.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index 411f280..0d378ad 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -646,7 +646,7 @@ int lookup_read_map(struct autofs_point *ap, time_t age, void *context)
 	mc = source->mc;
 
 	if (source->recurse)
-		return NSS_STATUS_UNAVAIL;
+		return NSS_STATUS_TRYAGAIN;
 
 	if (source->depth > MAX_INCLUDE_DEPTH) {
 		error(ap->logopt,
